koku/koku/cache.py

```diff
@@ -93,6 +94,9 @@ def invalidate_view_cache_for_tenant_and_source_type(schema_name, source_type):
         )
     elif source_type in (Provider.PROVIDER_AZURE, Provider.PROVIDER_AZURE_LOCAL):
         cache_key_prefixes = (AZURE_CACHE_PREFIX, OPENSHIFT_AZURE_CACHE_PREFIX, OPENSHIFT_ALL_CACHE_PREFIX)
+    elif source_type in (Provider.PROVIDER_GCP, Provider.PROVIDER_GCP_LOCAL):
+        # TODO: Covert this back to a tuple, after we get more than one element.
+        cache_key_prefixes = [GCP_CACHE_PREFIX]
```

Comment:
I think we can just make this a tuple and extend it or adjust it as needed.

```suggestion
        cache_key_prefixes = (GCP_CACHE_PREFIX,)
```

Reply:
ahhhhhhhh. Thanks for this @blentz, I was trying to figure out why indexing was different based on if a tuple has a single element in it or not. I was missing the comma.

Example:
```
>>> single_tup = ("Cody")
>>> for idx in single_tup:
...     print(idx)
...
C
o
d
y
>>> single_tup_comma = ("Cody",)
>>> for idx in single_tup_comma:
...     print(idx)
...
Cody
```
This is a better solution and I learned something new. 🙇